cardform/src/main/res/drawable/cf_icon_close.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#696969"
```

Comment:
what do you think about adding color by config file?

Reply:
Done!